app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
+      $('#request_user_attributes_library_id').focus();
+  }, 1);
+
+  $('form').on('keypress', function(e) {
+    if(e.which == 13) {
+      $('.form-group input').checkForm(e);
```

Comment:
'.form-group input' is an array of objects. I suppose I don't really need to specify input, but I guess I don't see what you would want to validate with this script that is not a user input, since the reason for the script is to check whether they have values for (some of) the user inputs.